### PR TITLE
Add in item_equipment.sql for main hand or no MID items.

### PR DESCRIPTION
**Pull request by helixhamin**
Still need to find MIDs for all the other items missing before I start adding (Not currently included in this PR, for a future PR). Added some of the missing item mods, but I want to limit the amount of changes in the submission, so I will add more mods and latents to the next PR.

**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Adds missing items and some of their latents/mods/etc, to item_equipment and item_weapon

## Steps to test these changes

Update the SQL with the tables, use gm commands to additem, see if items added to equipment can be equipped, items added to weapon can we weaponed, and items added to mods can mod stats.

### CODE REVIEW

**Comment by helixhamin** [comment]
~~Hmm, not sure why the "build / Full_Startup_Checks_Linux (pull_request)" failed. I only modified SQL, which shouldn't cause this, and didn't submit anything that changed it.~~

nvm, it just needed to run again once the process was free xD

**Review by Xaver-DaRed — APPROVED** [review]

File: sql/item_mods.sql [hunk 1]
```diff
@@ -2109,7 +2109,7 @@ INSERT INTO `item_mods` VALUES (10615,384,300); -- HASTE_GEAR: 300
 
 -- Rheic Schuhs
 INSERT INTO `item_mods` VALUES (10616,1,26);      -- DEF: 26
-INSERT INTO `item_mods` VALUES (10616,161,-3000); -- DMGPHYS: -3000
```
Comment (TeoTwawki) [inline sql/item_mods.sql:2112]: _eyes suspiciously and consults git-blame_

**Review by TeoTwawki — APPROVED** [review]

**Comment by helixhamin** [comment]
Sorry, think I messed up my commit. I did switch/checkout, under a new branch name, but I guess that it didn't push to the repository, and pushed all my new commits to the same PR, instead of a separate PR.
Sorry, not sure what to do next to resolve.

The new ones are from @vitus0123456 in https://github.com/LandSandBoat/server/discussions/3839. I had to correct some stuff, and added some stuff, but they did most of the work on the latest.

**Comment by helixhamin** [comment]
Okay, I broke it...
No idea what to do now...

**Comment by helixhamin** [comment]
Closing. I'll fix whatever my issues are and try again.

**Pull request closed without merging**